hibike/devices/ExampleDevice/example_device.cpp

```diff
@@ -1,5 +1,5 @@
 #include "example_device.h"
-
+#include "Arduino.h" //make sure to explicitly get the arduino api.
```

Comment:
There should still be a blank line between includes and the comment on the next line